ext/xml/tests/xml_parser_get_option_variation4.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+xml_parser_get_option() - Test that a parameter not set defaults
+and returns a E_WARNING-level message with "Unknown option" text
+--CREDITS--
+Symeon Charalabides <symeon@systasis.com> - @phpdublin
+--SKIPIF--
+<?php
+if (!extension_loaded('xml')) {
+    exit('Skip - XML extension not loaded');
+}
+?>
+--FILE--
+<?php
+
+$xmlParser = xml_parser_create();
+
+var_dump(xml_parser_get_option ($xmlParser, XML_OPTION_SKIP_WHITE));
+
+?>
+--EXPECTF--
+Warning: xml_parser_get_option(): Unknown option in %s on line %d
```

Comment:
Oh, I think that should actually be a ValueError instead of a warning. That still could be changed later, though.

Reply:
The source code throws E_WARNING, so that needs to be tested at the moment. I'm reluctant to touch .c files out of self-preservation.

Reply:
Fair enough. :)